### PR TITLE
update system methods.

### PR DESCRIPTION
**Pull request by Jim8y**

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs [hunk 1]
```diff
@@ -59,6 +59,12 @@ static Instruction()
             }
         }
 
+        internal Instruction AddTarget(JumpTarget target)
+        {
+            target.Instruction = this;
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs:64]: This doesn't add the NOP, isn't it?

**Review by shargon — COMMENTED** [review]
You also moved some methods and it complicate the review :(

**Comment by Jim8y** [comment]
> You also moved some methods and it complicate the review :(

I know,,, i made this super huge pr,,,,,, it just keep growing while i was trying to make it work....... have being working on this for 15 hours without resting......

File: src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs (hunk 1 above)
Comment (Jim8y) [reply src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs:64]: This one will not, we can set jump target to existing opcode, if there is none, we can add a nop and attach the jumptarget to it.. This one inside the Instruction is for adding jump target to existing opcode, while the one with the same name `AddTarget` is for situations where we have to set jumptarget to Nop.

I made this change to make it easier to write the code, and make instructoins look shorter.....

**Comment by Jim8y** [comment]
I tried my best to make it easier for reviewing, and avoided changes to the compile result.....

BUT, there is still a case that will change the artifact,,, which is I added `exception message`  while calling `throw` for nemuric methods.

File: src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert/Instruction.cs:64]: But it's working, isn't it?

**Comment by Jim8y** [comment]
this will not be merged.

**Pull request closed without merging**